### DISCUSSION TITLE
error:controller crashed

### DISCUSSION THREAD

**Discussion by ghost** [cyberbotics/webots#4547] [Q&A]
When I start the simulation，windows reported that .exe failed to start(0xc0000279),and the console information is as follows
<img width="853" alt="webots" src="https://user-images.githubusercontent.com/96941078/169641839-2642b35e-860a-423e-b747-e263248b3748.png">
:

**Comment by omichel** [comment]
Which version of Webots are you using?

**Reply by ghost** [reply]
2021a，and 2020a has the same situation.

**Reply by omichel** [reply]
I would recommend you to try R2022a instead.

**Reply by ghost** [reply]
Thank you,I'll have a try.

**Reply by ghost** [reply]
It seems that the problem is still unresolved.It may be the problem of computer configuration, such as the lack of some necessary library files？ But there has been no problem before，I will try to uninstall some system updates later.
